geode-connectors/src/acceptanceTest/java/org/apache/geode/connectors/jdbc/JdbcDistributedTest.java

```diff
@@ -779,6 +882,40 @@ private void createRegionUsingGfsh() {
     gfsh.executeAndAssertThat(createRegionCmd.toString()).statusIsSuccess();
   }
 
+  private void createRegionUsingGfshForGroup(boolean isAccessor, String groupName) {
```

Comment:
We'd better explicitly mention `replicated` region in the method name. Since the other helper method explicitly mentions `Partition` in the method name `createPartitionRegionUsingGfshForGroup`

Reply:
The convention in this test is to use createRegionUsingGfsh vs createPartitionRegionUsingGfsh. I did not change or enhance it.